kola-denylist.yaml

```diff
@@ -35,6 +35,18 @@
 
 - pattern: ext.config.shared.var-mount.luks
   tracker: https://github.com/openshift/os/issues/1656
+  osversion:
+    - c9s
+    - rhel-9.6
```

Comment:
```suggestion
    - rhel-9.4
    - rhel-9.6
```

Reply:
I saw this failing also at [rhel-9.4](https://github.com/openshift/os/issues/1656)